README.md

```diff
@@ -94,40 +94,60 @@
+| Cryptography  | General cryptography classes including ECC implementation.                                        |
+| IO            | Data structures used for caching and collection interaction.                                      |
+| Ledger        | Classes responsible for the state control, including the `MemoryPool` and `Blockchain` classes.   |
+| Network       | Peer-to-peer protocol implementation classes.                                                     |
+| Persistence   | Classes used to allow other classes to access application state.                                  |
+| Plugins       | Interfaces used to extend Neo, including the storage interface.                                   |
+| SmartContract | Native contracts, `ApplicationEngine`, `InteropService` and other smart-contract related classes. |
+| VM            | Helper methods used to interact with the VM.                                                      |
+| Wallet        | Wallet and account implementation.                                                                |
+
+## Downloads
+
+Below are the available downloads for our project. Please select the appropriate file for your needs.
+
+### NEO CLI Node
+| Platform    | File                    | Download                                                                                           | SHA256                                                                                              |
+|-------------|-------------------------|----------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
+| Linux       | neo-cli-linux-x64.zip   | [V3.6.2](https://github.com/neo-project/neo-node/releases/download/v3.6.2/neo-cli-linux-x64.zip)   | 8195918E942D2B3BF5BD0ACC8A29157197F4594EC484E1AAC719E6605E4EF299                                    |
+| Windows     | neo-cli-win-x64.zip     | [V3.6.2](https://github.com/neo-project/neo-node/releases/download/v3.6.2/neo-cli-win-x64.zip)     | 4CFDD39DD7C0D37940BBBC4D42403599FFF96D4A8F9A8D394BB9A07400CAB47D                                    |
+| Mac OS      | neo-cli-osx-x64.zip     | [V3.6.2](https://github.com/neo-project/neo-node/releases/download/v3.6.2/neo-cli-osx-x64.zip)     | D0A5A50751353E22886C11EE4F8AE6C3DDBDE7EA090FE7B58A3064BAB3C8950D                                    |
+| Mac OS ARM  | neo-cli-osx-arm-x64.zip | [V3.6.2](https://github.com/neo-project/neo-node/releases/download/v3.6.2/neo-cli-osx-arm-x64.zip) | CAC83365F9E9E24F20BEB98C1BB9AD1FD9240E0A28F1D5DDDE0C58E5AA8D49F3                                    |
+| Portable    | neo-cli-portable.zip    | [V3.6.2](https://github.com/neo-project/neo-node/releases/download/v3.6.2/neo-cli-portable.zip)    | D268777F2A6C7E801E8FC3856BE72D336E0E2B6623BD76F2F8816D3A0F00D1D8                                    |
+
+### NEO Modules
```

Comment:
It's OK to mention them as a repo while they're separate (maybe link devpack as well, if it's not linked yet), but maintaining this list here is just duplication to me (and it will get out of sync).